### ISSUE TITLE
malsup.com

### ISSUE THREAD

**Issue by smed79** [lightswitch05/hosts#428]
Potential false positive.

`malsup.com`

Main domain and all sub are blocked in [ads-and-tracking-extended.txt](https://raw.githubusercontent.com/lightswitch05/hosts/master/docs/lists/ads-and-tracking-extended.txt)

@malsup FYI.

**Comment by lightswitch05** [comment]
Yeah, this one looks like a major false positive. I wonder how it got added in the first place.

Just FYI, this project is kinda in limbo at the moment. I had some tools to help with automation and keeping notes on domains. I had to do a system migration recently- and I haven't got around to doing anything with this project. So I don't have access to my notes. I'm not sure if I'm going to do anything with this project or not. The Pihole has been working on new list features that I'm kinda interested in taking advantage of- but those would not be 'hosts' format. On the other hand, I haven't added anything new to these lists in some time and it might just be better to let it die then to do the work to change formats only to continue not making it better. I haven't come to any decisions yet. If you find these lists useful, then I'm glad to hear it, but keep in mind they will only get worse with time without a lot of active maintenance.

**Comment by smed79** [comment]
Lists are useful, it's so sad to hear that we are losing maintenance of them.

---
Ping @funilrys @mitchellkrogza @StevenBlack @hagezi @nextdns
